app.rb

```diff
@@ -0,0 +1,136 @@
+require_relative './person'
+require_relative './rental'
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  # Code to list all books
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224444788-56e9d3ae-4eb6-4440-8a57-c63e09a2a5cc.png)

- [ ] Kindly remove those links from your Table of Contents section to make your readme file descriptive and professional 😊